husky_directory/templates/index.html

```diff
@@ -1,11 +1,100 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <title>UW Faculty/Staff/Student Directory</title>
-</head>
-<body>
-<h1>UW Directory</h1>
-Nothing here to see yet . . .
-
-</body>
+
+<html lang="eng">
+    <head>
+        <meta content="IE=edge" http-equiv="X-UA-Compatible"/>
+        <title>UW Faculty/Staff/Student Directory</title>
+        <meta http-equiv="content-type" content="text/html;charset=ISO-8859-1" />
+        <meta name="viewport" content="user-scalable=no, initial-scale=1.0, maximum-scale=1.0" />
+        <meta http-equiv="Cache-Control" content="no-cache" />
+        <meta http-equiv="Pragma" content="no-cache" />
+        <meta http-equiv="Expires" content="0" />
+        <meta name="googlebot" content="noarchive" />
+        <meta name="robots" content="noarchive" />
+
+        <!-- Desktop -->
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.css" type="text/css" />
+        <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/mini.css') }}">
+        <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/body.css') }}">
+
+        <link rel="stylesheet" type="text/css" media="print" href="{{ url_for('static', filename='css/print.css') }}">
+
+        <style>
+         .btn.search:hover { color: black; }
+        /*------------ Workday changes --------- */
+        ul.dir-listing { list-style-type: none; padding: 0; }
+        ul.multiaddr, ul.title, ul.address { list-style-type: none; padding: 0; }
+        li.dir-boxstuff { padding-bottom: .6em }
+        /*------------ End Workday stuff-------*/
```

Comment:
Can you explain why these changes are required? (Or was this code copied from elsehwere?)

Reply:
This is pre-existing code from the index.html in the old uw directory.